Code/Framework/AzCore/AzCore/std/allocator_traits.h

```diff
@@ -107,6 +107,18 @@ namespace AZStd
             using type = typename Allocator::difference_type;
         };
 
+        //! align_type
+        template<typename Allocator, typename AlignType, typename = void >
+        struct get_align_type
+        {
+            using type = typename std::make_unsigned<AlignType>::type;
```

Comment:
`AZStd::make_ununsigned_t` should be used here.

Reply:
Fixed.